lib/auth/auth.go

```diff
@@ -347,7 +347,14 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 	if cfg.ProvisioningStates == nil {
 		cfg.ProvisioningStates, err = local.NewProvisioningStateService(cfg.Backend)
 		if err != nil {
-			return nil, trace.Wrap(err, "Creating provisioning state service")
+			return nil, trace.Wrap(err, "creating default provisioning state service")
+		}
+	}
+	if cfg.IdentityCenter == nil {
+		svcCfg := local.IdentityCenterServiceConfig{Backend: cfg.Backend}
+		cfg.IdentityCenter, err = local.NewIdentityCenterService(svcCfg)
+		if err != nil {
+			return nil, trace.Wrap(err, "creating default identity center service")
```

Comment:
Do we really need to mention `creating default provisioning state service` and `creating default identity center service` here? `default identity center service` is not clear enough and is not specific in mentioning if this is on auth service initialization or plugin runtime. Maybe mention this is the local service initialization in auth bootstrap or remove if altogether (other services does not write any extra error message)